tests/test_e2e_41_kytos_auth.py

```diff
@@ -0,0 +1,193 @@
+import os
+import time
+import shutil
+import requests
+from tests.helpers import NetworkTest
+from kytos.core.auth import UserController
+
+CONTROLLER = '127.0.0.1'
+KYTOS_API = 'http://%s:8181/api/kytos' % CONTROLLER
+
+
+class TestE2EKytosAuth:
+    net = None
+
+    def setup_method(self, method):
+        """
+        It is called at the beginning of every class method execution
+        """
+        self.net.start_controller(clean_config=True, enable_all=False)
+        self.username = "Superuser"
+        self.password = "Password123"
+        self.email = "user@kytos.io"
+        self.user_controller = UserController()
+        self.user_controller.create_user({
+            "username": self.username,
+            "password": self.password,
+            "email": self.email
+        })
+        self.token = self.get_token()
+        self.auth_header = {"Authorization": f"Bearer {self.token}"}
+        time.sleep(5)
+
+    @classmethod
+    def setup_class(cls):
+        cls.net = NetworkTest(CONTROLLER)
+        cls.net.start()
+        # rotate logfile (copy/truncate strategy)
+        logfile = '/var/log/syslog'
```

Comment:
@Alopalao, could you double check if we needed this `shutil.copy` here? Looks like this was only meant for the startup test suite.

Reply:
Yes, it is not necessary for these tests.